package.json

```diff
@@ -14,8 +14,14 @@
     "start": "webpack serve --config webpack/webpack.dev.ts",
     "fetch-deps": "ts-node ./util/fetch_deps.ts",
     "tsc-no-emit": "tsc --noEmit",
-    "lint": "eslint --max-warnings=0 ./",
-    "lintfix": "eslint --max-warnings=0 --fix ./",
+    "lint": "run-s lint:**",
+    "lint:dprint": "dprint check",
+    "lint:dprint:trigger": "dprint --config=dprint.trigger.json check",
+    "lint:eslint": "eslint --max-warnings=0 ./",
```

Comment:
Do we need to teach VSCode about dprint as well? Currently it knows about eslint and so there's errors when things are formatted incorrectly.  Is it possible to do this with dprint too?

Reply:
I didn't using vscode for a lont time.....

Reply:
And I think dprint extension of vscode doesn't support multiple config file either.

Reply:
In this case I suggest we drop 300 line width in triggers files as we no longer need multiple line for netRegex/netRegexCn/netRegexKo and so on and in the future will replace netRegex lone line with it's param.

Reply:
Yeah, I was having a similar thought.  Let me look through scripts to make sure nothing is trying to parse trigger files as text and will break if things go on different lines.

Reply:
I tried `yarn util findTranslations` and it works fine, don't know if any other script may break.

Reply:
I know `.github/scripts/auto-label.cjs` may break if we add parentheses to these value that got splited in new line...

Reply:
Yeah, I looked through everything, and it seems like everything is doing a dynamic `import` of trigger files these days and not parsing as text (phew).  So, I think this should be ok.

Can you explain your concern about `auto-label`? It seems like it should be ok to me.

Reply:
And we can write a formatter by our own but I think it's too much... https://github.com/dprint/js-formatter

Reply:
> 

we are actually using json5 to parse the ast result into js object to avoid executing any code, adding parentheses breaks that.

Reply:
Why are there parentheses? I don't understand.

Reply:
```ts
    {
      'locale': 'fr',
      'replaceSync': {
        'This humanoid prototype can perfectly replicate': (
          'Ce modèle exceptionnel est basé sur les non moins exceptionnelles divinités guerrières'
        ),
        'Utilizing our data on Sophia': (
          'Quoi de mieux pour dominer les divinités guerrières que de retourner leur puissance contre elles?♪',
         )
      },
```

if someone do line break like this (

Reply:
Oh, I see.  But this could already be an issue today, right?

What about an extra test that the section is valid json5 so that it's obvious when this happens?

Reply:
Also, if you don't mind, could you update the line length to 100 in a separate PR, land that first, and then land this patch? I feel like changing the line length will be very large, and it will make this PR very unwieldy to look at further changes.

Reply:
Yes, but I don't think that's very necessary, let's do that when this actually happen ...